Add/Update community tools (#13)
vtex-apps/store-form: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/store-form) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
> [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
> <!-- ALL-CONTRIBUTORS-BADGE:END -->
> 
> # Store Form
> 

Reviewer: ```suggestion
# Store Form

<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

📢 Use this project, [contribute](https://github.com/vtex-apps/store-form) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
# Store Form

> <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
> [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
> <!-- ALL-CONTRIBUTORS-BADGE:END -->

Reviewer: ```suggestion
```
